.github/workflows/dependencies.yml

```diff
@@ -13,19 +13,24 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: 10.17.0
-      - name: Install Tools
+      - name: Install
         run: |-
-          npm -g install lerna npm-check-updates@^9.0.0
```

Comment:
Moved this as a dependency of the root `package.json` so that its also easily available locally.